src/elements/content-preview/ContentPreview.js

```diff
@@ -184,6 +186,34 @@ const LoadableSidebar = AsyncLoad({
 });
 
 class ContentPreview extends React.PureComponent<Props, State> {
+    static defaultProps = {
```

Comment:
When I formatted the file it moved this to the top automatically so I assume some where in our configuration its requiring components to have `defaultProps` at the top

Reply:
Has this file actually changed apart from formatting? It looks like the logical changes have been removed?